kotlin/android/app/build.gradle.kts

```diff
@@ -103,6 +116,14 @@ android {
             )
 
             resValue("string", "app_name", "\"Firezone\"")
+
+            firebaseAppDistribution {
+                serviceCredentialsFile = System.getenv("FIREBASE_CREDENTIALS_PATH")
+                artifactType = "AAB"
+                releaseNotes = "https://github.com/firezone/firezone/releases"
+                testers = System.getenv("FIREBASE_APP_TESTERS")
```

Comment:
Can we use groups instead of individual tests here? 
https://firebase.google.com/docs/app-distribution/android/distribute-gradle#step_3_configure_your_distribution_properties

Reply:
Good call, updated!